package/gluon-core/luasrc/lib/gluon/upgrade/140-firewall-rules

```diff
@@ -1,25 +1,87 @@
 #!/usr/bin/lua
+-- TODO: replace REJECT by DROP. only use REJECT for debugging
 
 local site = require 'gluon.site_config'
 local uci = require('simple-uci').cursor()
 
+uci:section('firewall', 'zone', 'wan', {
+	name = 'wan',
+	network = { 'wan','wan6',},
+	input = 'REJECT',
+	output = 'ACCEPT',
+	forward = 'REJECT',
+	mtu_fix = '1',
+	conntrack = '1',
+	log = '1',
+}
+)
```

Comment:
The 'wan' zone is unnamed by default, therefore we can't reference it using uci:section(). We should just keep the current code for handing the WAN zone.